### PR TITLE
adding fftmonitor

### PR DESCRIPTION
**Pull request by bryanbrady**

### CODE REVIEW

File: python/sbp/client/util/fftmonitor.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+printing them out.
+"""
+
+# from sbp.client.drivers.pyserial_driver import PySerialDriver
+
+from collections import defaultdict
+from sbp.client.drivers.network_drivers import TCPDriver
+from sbp.client import Handler, Framer
+from sbp.gnss import GPSTimeSec
+try:
```
Comment (bryanbrady) [inline python/sbp/client/util/fftmonitor.py]: This needs to be fixed.

File: python/sbp/client/util/fftmonitor.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+            self.incomplete_ffts[channel][timestamp]['frequencies'],
+            frequencies,
+            axis=0)
+
+        self.incomplete_ffts[channel][timestamp]['amplitudes'] = np.append(
+            self.incomplete_ffts[channel][timestamp]['amplitudes'],
+            amplitudes,
+            axis=0)
+
+        if len(self.incomplete_ffts[channel][timestamp]['frequencies']) == 512:
```
Comment (axlan) [inline python/sbp/client/util/fftmonitor.py]: Is this 512 available as a defined constant somehwhere?

File: python/sbp/client/util/fftmonitor.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+            current_fft = self.incomplete_ffts[channel].pop(timestamp)
```
Comment (axlan) [inline python/sbp/client/util/fftmonitor.py]: What's the expected behavior if you miss a message?

Comment (bryanbrady) [reply python/sbp/client/util/fftmonitor.py]: Ignore it and wait for the next one.

File: python/sbp/client/util/fftmonitor.py (hunk 2 above)
Comment (bryanbrady) [reply python/sbp/client/util/fftmonitor.py]: If it is, I haven't found it. Probably should be, though.  It was hardcoded in the Spectrum Analyzer in the Console

**Pull request merged**